### PR TITLE
Add Cory's edits to public folders article

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #1646

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-60ad0cc774/how-to/).

**Comment by kmsholcomb** [comment]
Hi @DonNateR. My edits are in the attached docs. I took a stab at simplifying some of the steps, which were overly detailed. Please let me know if you have any questions. Thanks!

[Manage public folders in the control panel for Hosted Exchange 2013 and 2016_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/628269/Manage.public.folders.in.the.control.panel.for.Hosted.Exchange.2013.and.2016_editsKH.pdf)
[Manage public folders in the control panel for Hosted Exchange 2013 and 2016_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/628270/Manage.public.folders.in.the.control.panel.for.Hosted.Exchange.2013.and.2016_editsKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0e72bec18a/how-to/)

File: content/exchange/manage-public-folders-in-the-control-panel-for-hosted-exchange-2013.md [hunk 1]
```diff
@@ -58,21 +56,15 @@ steps:
 
 1. Click **Create Folder**.
 
-   A popup dialog box is displayed.
+2. The the popup dialog box, enter a name for the folder.
```
Comment (catlook) [inline content/exchange/manage-public-folders-in-the-control-panel-for-hosted-exchange-2013.md]: double "the"

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fef3c259df/how-to/)

**Pull request merged**